addon/components/bs-navbar/link-to.hbs

```diff
@@ -0,0 +1,14 @@
+<BsLinkTo
+  @route={{this.route}}
+  @model={{this.model}}
+  @models={{this.models}}
+  @query={{this.query}}
+  @replace={{@replace}}
+  @disabled={{@disabled}}
+  @current-when={{@current-when}}
+  @activeClass={{@activeClass}}
```

Comment:
I don't think `<BsLinkTo>` supports `@replace`, `@current-when` and `@activeClass`. I guess a left-over from an earlier iteration.

Reply:
Good catch, will fix!